observability/observability.go

```diff
@@ -5,6 +5,8 @@ import (
 	"net/http"
 	"sort"
 
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
this is only here so that I can type Alias Collector as prometheus.Collector. Later we can change this out to an actual interface when we have more information on how observability will work with other providers.